developer_documentation/contribution_guidelines.Rmd

```diff
@@ -91,6 +92,89 @@ the devil].
+    where `type` is either `hotfix` or `release`, and version is the new version
+    number. For example, `hotfix-v3.0.3` would be a hotfix branch that
+    changes the version number to `3.0.3`. Feature branch names should reflect
+    their purpose, and can be anything other than `master`, `develop`,
+    `hotfix-*`, or `release-*`.
+
+  - Whenever the package version changes, a description of the related changes
+    should be added to the changelog in `NEWS.md` as a new section titled with
+    the new version number. Such version changes will happen via release and
+    hotfix branches. Release branches will generally include changes made during
+    the course of several feature branches, so they may require a significant
+    amount of documentation in `NEWS.md`. To make it easier to prepare a list of
+    these changes, each feature branch should include a description of its own
+    changes in an `UNRELEASED` section of `NEWS.md`. For more information about
+    updating the changelog, please see the comment at the top of `NEWS.md`.
+
+* The following is a short description of this branching model:
+
+  - The `main` branch always contains the latest stable release of the BioCro
+    GitHub repository. In fact, a new tag and release is made any time changes
+    are merged into `main`, and such changes should always be accompanied by an
+    increment to the BioCro R package version number.
+
+  - The `develop` branch contains bug fixes and new features that have been
+    completed but may not have been released yet.
+
+  - Hotfix branches are for urgent changes that warrant immediate incorporation
+    into `main`; typically these are bug fixes. A hotfix branch should be
+    branched off `main`. When ready, it should first merged into `main` via an
+    approved pull request. Then, a second pull request should be made to merge
+    into `develop`. If there are no merge conflicts or test failures, this
+    second request can be merged without any additional approval. When both
+    merges are complete, the hotfix branch should be deleted.
+
+  - Feature branches are for less urgent changes that do not require immediate
+    release; for example, the addition of a new R function to the package
+    namespace. A feature branch should be branched off `develop` and merged
+    back into `develop` via an approved pull request. When the merge into
+    `develop` is complete, the feature branch should be deleted.
+
+  - When sufficiently many changes have accumulated in `develop` to justify a
+    new version of the package, a release branch is used to move the changes
+    from `develop` to `main`. Release branches should not include substantive
+    changes; rather, a release branch is primarily used to increment the package
+    version and to ensure an up-to-date changelog in `NEWS.md`. A release branch
+    should be branched off `develop`. When it's ready, it should first merged
```

Comment:
Maybe after "A release branch should be branched off `develop`." add "Update NEWS.md to include an # UNRELEASED section that describes the changes that will be introduced in the feature branch (see NEWS.md for more details)."